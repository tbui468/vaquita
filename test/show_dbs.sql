create database sol;
show databases;
drop database sol;
exit;
