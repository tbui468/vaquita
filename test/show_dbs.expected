created database sol
databases
sol
dropped database sol
